### DISCUSSION TITLE
[FeatureRequest] Consider allowing a default datasource per datasource "type"

### DISCUSSION THREAD

**Discussion by tomwilkie** [grafana/grafana#38336] [Data Sources]
**What would you like to be added**:

The ability to specify a different default datasource for each kind/type of datasource.

**Why is this needed**:

My understanding is Grafana can only have one default datasource.  And I try and template out the datasource in all my dashboards to make them more reusable.  The templates always specify a type.  I think being able to have a default per type would make this more flexible.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
